src/main/java/uk/gov/hmcts/reform/slc/logging/AppDependency.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.slc.logging;
+
+final class AppDependency {
+
+    /**
+     * Service bus message receiver dependency.
+     * Used to track dependency of how long in millis did it take to...
+     */
+    static final String MESSAGE_RECEIVER_DEPENDENCY     = "MessageReceiver";
+
+    /**
+     * .. receive message receive from service bus queue.
+     */
+    static final String MESSAGE_RECEIVED_COMMAND        = "MessageReceived";
+
+    /**
+     * .. release lock and remove message from service bus queue.
+     */
+    static final String MESSAGE_COMPLETED_COMMAND       = "MessageCompleted";
```

Comment:
can names of dependencies be separate from names of commands?

Reply:
also those suffixes won't be needed then

Reply:
Can be. Not sure I follow you though. Grouping of variables inside the class or introducing local (external) classes to group dependency activity?